test/mpi/test_mpi.cc

```diff
@@ -72,13 +90,22 @@ UccTestMpi::UccTestMpi(int argc, char *argv[], ucc_thread_mode_t _tm, int is_loc
     UCC_CHECK(ucc_context_config_read(lib, NULL, &ctx_config));
     UCC_CHECK(ucc_context_create(lib, &ctx_params, ctx_config, &ctx));
     ucc_context_config_release(ctx_config);
+    setenv("UCC_TL_UCP_TUNE", "alltoall:0-inf:@onesided", 1);
```

Comment:
setenv/unsetenv will alter user setting. Need to do getenv first. If not NULL save it, then restore it back. This is tmp solution. We should implement config modify for TL and use that (as separate PR)